src/test.props

```diff
@@ -8,5 +7,8 @@
 -->
   <PropertyGroup Condition=" $(MSBuildProjectName.EndsWith('.Tests')) AND '$(BuildingInsideVisualStudio)' == 'true' ">
     <OutputPath>$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.proj))\\bin\\tests\\Windows_NT.$(Platform).$(Configuration)\$(MSBuildProjectName)\dnxcore50</OutputPath>
-  </PropertyGroup>  
+  </PropertyGroup>
+  <PropertyGroup Condition="$(MSBuildProjectName.EndsWith('.Tests'))">
+    <XunitOptions>-parallel none</XunitOptions>
```

Comment:
There is a bug either in xunit or dotnet core where running more than one test at a time causes a race condition. The threads will get stuck in the call to send the HTTP request to the bridge. Even when I put a lock around the methods so only one thread is making an HTTP request at a time, it will still get stuck indefinitely on the one thread trying to make the request. Perhaps an alternative is to only apply this option when running OuterLoop tests.